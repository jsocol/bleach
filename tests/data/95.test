<div id="24">1<set/xmlns=`urn:schemas-microsoft-com:time` style=`behAvior:url(#default#time2)` attributename=`innerhtml` to=`<img/src="x"onerror=alert(24)>`>//["'`-->]]>]</div>
