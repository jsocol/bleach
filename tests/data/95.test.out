<div id="24">1`&gt;//["'`--&gt;]]&gt;]</div>
